Cython/Utility/Synchronization.c

```diff
@@ -359,6 +370,26 @@ static void __Pyx__Locks_PyThreadTypeLock_Lock(__Pyx_Locks_PyThreadTypeLock lock
 #define __Pyx_Locks_PyMutex_LockGil(l) PyMutex_Lock(&l)
 #define  __Pyx_Locks_PyMutex_LockNogil(l) PyMutex_Lock(&l)
 
+// For Python 3.13, we need to check if PyMutex_IsLocked is available
+#if PY_VERSION_HEX >= 0x030e00C1
+// Python 3.14+ has PyMutex_IsLocked  
+static CYTHON_INLINE int __Pyx_Locks_PyMutex_Locked(__Pyx_Locks_PyMutex lock) {
+    return PyMutex_IsLocked(&lock);
+}
+#else
+// Python 3.13: Use internal implementation
+// The _Py_LOCKED constant is defined in internal/pycore_lock.h
+#ifndef _Py_LOCKED
+#define _Py_LOCKED 1
```

Comment:
It looks to be in https://github.com/python/cpython/blob/26c2a49857a568c62c4f0be2bd01501289743b63/Include/cpython/lock.h#L6 so you shouldn't need to define it